Source/WebCore/dom/Observable.idl

```diff
@@ -40,4 +41,8 @@ interface Observable {
   [CallWith=CurrentScriptExecutionContext] Observable take(unsigned long long amount);
 
   [CallWith=CurrentScriptExecutionContext] Observable drop(unsigned long long amount);
+
```

Comment:
Let's see, there may be conflicts with https://github.com/WebKit/WebKit/pull/32099. But should be okay, Ill resolve resolve with whatever lands first.